tests/test_streaming.py

```diff
@@ -8,6 +8,7 @@
 import pytest
```

Comment:
Same here. Just test the `bytes_to_int()` and `number_abbrev_to_int()` with various parameter as part of test_util.py !!